server/src/engine/socket.ts

```diff
@@ -85,60 +84,39 @@ export function isClosed() {
 let lastManualStopTimestamp: number = 0
 
 export function initialiseSocket({ uri, onOpen, onClose }: InitialiseSocketInput) {
-  if (!uri) {
-    throw 'Must be called with an uri'
-  }
-  webSocket = new WebSocket(uri)
+  webSocket = new ReconnectingWebSocket(uri, [], {
+    WebSocket,
+    maxRetries: 5,
```

Comment:
This is not good right. This will break after a few standby's. It should be infinite.

Reply:
In fact, I would worry whether ReconnectingWebSocket supports that.